projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.scss

```diff
@@ -1,21 +1,71 @@
 @import '/projects/commudle-admin/src/assets/styles/variables';
+
 .external-feed {
-    width: 100%;
-  
+  .left-column {
+    width: $small-column-width;
+
     .header {
+      border: $border-1 solid $solitude;
+      border-radius: $border-16;
```

Comment:
this should be 10 (just like on home page or lab show page

Reply:
Done in all the cards